_rules/table-headers-attribute-refer-to-data-cells-a25f45.md

```diff
@@ -30,7 +30,9 @@ This rule applies to any `headers` attribute specified on a [`cell`][] within a
 
 ## Expectation
 
-Each target attribute is [a set of space separated IDs][], each of which is the ID of an element, that is a [`cell`][] of the same [`table`][] as the target element.
+Each target attribute is [a set of space separated IDs][], each of which is the ID of an element, that is a [`cell`][] of the same [`table`][] as the target element different from the target.
```

Comment:
I think this should be handled in a separate expectation.

Reply:
Yes, that will be easier to read 👍